data/core/macros/abilities.cfg

```
@@ -146,6 +146,8 @@ This ability will not cure an affected unit of poison, however, only delay its e
         female_name= _ "female^diversion"
         description= _ "If there is an enemy of the target on the opposite side of the target, this unit diverts the target’s attention and reduces its chance to hit by 20%."
         special_note=_"If there is an enemy of the target on the opposite side of the target while attacking it, this unit diverts the target’s attention and reduces its chance to hit."
+        name_affected= _ "<span color='#DD6F6F'>accuracy : -20%</span>"
```

```suggestion
        name_affected= _ "<span color='#DD6F6F'>accuracy: -20%</span>"
```
space before color not needed.

---

space removed